kong/init.lua

```diff
@@ -98,9 +99,95 @@ local ffi = require "ffi"
 local cast = ffi.cast
 local voidpp = ffi.typeof("void**")
 
+local plugins_map_version
+local plugins_map_semaphore
+
+local PLUGINS_MAP_CACHE_OPTS = { ttl = 0 }
+
+local configured_plugins
 local loaded_plugins
 local schema_state
 
+local function build_plugins_map(db, version)
+  local map = {}
+
+  for plugin, err in db.plugins:each() do
+    if err then
+      return nil, tostring(err)
```

Comment:
The new DAO guarantees that this value will be a string (no trick, ever, promise!). It is one of its design goals to get rid of such patterns like `tostring(err)` on the consumer-side (from the caller's POV). Can we remove the `tostring` wrap?

Reply:
This was a copy paste leftover from old code. Will fix.